vizro-ai/hatch.toml

```diff
@@ -58,8 +58,8 @@ build = "mkdocs build --strict"
 # Disable warnings on the linkcheck so that HTTP redirects are accepted. We could ignore only that warning and specify
 # more advanced settings using a linkcheckerrc config file. openai.com doesn't seem to work well with linkchecking,
 # throwing 403 errors, but these are not real errors.
-link-check = "linkchecker site --check-extern --no-warnings --ignore=404.html --ignore-url=127.0.0.1 --ignore-url=https://platform.openai.com/docs/models --ignore-url=openai.com --ignore-url=https://openai.com/"
-serve = "mkdocs serve --strict"
+link-check = "linkchecker site --check-extern --no-warnings --ignore=404.html --ignore-url=127.0.0.1 --ignore-url=https://vizro.readthedocs.io/ --ignore-url=https://platform.openai.com/docs/models --ignore-url=openai.com --ignore-url=https://openai.com/"
```

Comment:
Do we also need to ignore `https://vizro-ai.readthedocs.io/`?

Reply:
Hmm yes, I wasn't sure what to put here. That seems to just redirect to https://vizro.readthedocs.io/projects/vizro-ai/. What is the difference between these two? Which is our "recommended" URL?

Reply:
@antonymilne The recommended URL is `vizro.readthedocs.io/projects/vizro-ai/` since this indicates that `vizro-ai` is a subproject of `vizro` (following the same approach as Kedro/Kedro-Viz). I think/hope I've eliminated all use of `https://vizro-ai.readthedocs.io/` but maybe it's worth a check to make sure.

Reply:
I don't mind - up to you where to just stick it in this PR or do a separate one.

Reply:
I made a [commit with the change](https://github.com/mckinsey/vizro/pull/535/commits/a0ef59f2f53aa76e333624dae084ee4b0d536de8)...let's see what chaos that unleashes and revert it if it's too much to deal with here.